modules/docs/mdx/welcomePage.tsx

```diff
@@ -1,155 +1,173 @@
-        This project provides a set of components for the Workday Canvas Design System that can be
-        used to implement user experiences consistent with{' '}
-        <ExternalHyperlink href="https://canvas.workdaydesign.com/">
-          Workday Design Principles.
-        </ExternalHyperlink>
-      </Text>
-      <Heading size="medium" borderBottom={`1px solid ${colors.blueberry200}`} paddingBottom="xxs">
-        Quick Links
-      </Heading>
-      <Grid
-        gridAutoColumns="auto"
-        gridTemplateColumns="repeat(auto-fill, minmax(250px, 1fr))"
-        gridGap="s"
-      >
-        <Grid
-          as={Card}
-          gridTemplateRows="1fr 1fr 2fr 1fr"
-          depth="none"
-          display="grid"
-          maxHeight={320}
-        >
-          <SystemIcon
-            color={colors.cantaloupe400}
-            colorHover={colors.cantaloupe400}
-            icon={rocketIcon}
-            size={60}
-          ></SystemIcon>
-          <Card.Heading>Getting Started</Card.Heading>
-          <Card.Body>
-            <Text>For all things getting started including helpful guides and docs.</Text>
-          </Card.Body>
-          <Grid.Item
-            as={Hyperlink}
-            alignSelf="end"
-            marginTop="xs"
-            href="https://workday.github.io/canvas-kit/?path=/docs/guides-getting-started--docs"
-          >
-            {' '}
-            Getting Started Guide
-          </Grid.Item>
-        </Grid>
-        <Grid
-          as={Card}
-          gridTemplateRows="1fr 1fr 2fr 1fr"
-          depth="none"
-          display="grid"
-          maxHeight={320}
-        >
-          <SystemIcon
-            color={colors.blueberry400}
-            icon={tokensIcon}
-            colorHover={colors.blueberry400}
-            size={60}
-          ></SystemIcon>
-          <Card.Heading>Tokens</Card.Heading>
-          <Card.Body>
-            <Text>
-              Tokens are the smallest pieces of our Design System with the primary function of
-              storing UI information.
+    <div className="sb-unstyled">
+      <div className={parentFlexStyles}>
+        <Box cs={{overflow: 'hidden', position: 'relative'}}>
+          <Flex cs={{flexDirection: 'row', position: 'absolute', top: '45%', left: '10%'}}>
+            <Text typeLevel="title.medium" cs={textStyles}>
+              Canvas Kit
             </Text>
-          </Card.Body>
-          <Grid.Item
-            alignSelf="end"
-            as={Hyperlink}
-            marginTop="xs"
-            href="https://workday.github.io/canvas-kit/?path=/docs/tokens--overview"
+            <Text typeLevel="title.medium" cs={[textStyles, versionStyles]}>
+              v{version}
+            </Text>
+          </Flex>
+          <img
+            src={headerImage}
+            alt="banner with canvas kit logo and version"
+            className={imageStyles}
+          />
+        </Box>
+        <Text typeLevel="body.medium">
+          {' '}
```

Comment:
These forced spaces are probably not important.

Reply:
removed, ty